source/agora/consensus/protocol/Nominator.d

```diff
@@ -187,31 +264,61 @@ extern(D):
 
     /***************************************************************************
 
-        Try to begin a nomination round.
+        Gets the expected block nomination time for for the provided height.
+
+        Returns:
+            the expected block nomination time for the provided height
+
+    ***************************************************************************/
+
+    private ulong getExpectedBlockTime (Height height) @safe @nogc nothrow pure
```

Comment:
We should change all `time_t` to `ulong` since `time_t` doesn't seem to be portable on Windows.